clouds/clouds-aws/src/main/java/aws/core/identification/AwsRegion.java

```diff
@@ -0,0 +1,26 @@
+package aws.core.identification;
+
+
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import sunstone.api.inject.Hostname;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Annotation used for automatic resolution without providing any further details.
```

Comment:
I guess this one is not automatic as `AwsAutoResolve`

Reply:
Removed.